src/Tempest/View/src/Stubs/view.stub.php

```diff
@@ -1 +1,3 @@
+<?php
 
+declare(strict_types=1);
```

Comment:
broke phpstan somehow? 😅 should this be reverted?

Reply:
Yeah, let's ignore stub files. I'll do the change